docs/user/assets/telemetry-stages.drawio.svg

```diff
@@ -1,4 +1,268 @@
\ No newline at end of file
+<svg host="65bd71144e" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="721px" height="241px" viewBox="-0.5 -0.5 721 241" content="&lt;mxfile&gt;&lt;diagram id=&quot;xku7MT8EHSMA1KCECYjK&quot; name=&quot;Page-1&quot;&gt;7Vlbc+smEP41nmkfnBHC8uXRsuNzMtNOTpOHkz5iCUucYKEi5Et/fUFCsiRw6tT2iTsTv1gsLNr9vmUXUA/O1rsvHKXx7yzEtOc64a4H5z3XBQ6YyD8l2ZeSwXhUCiJOQj3oIHgmf+NKU0tzEuKsNVAwRgVJ28KAJQkOREuGOGfb9rAVo+23pijChuA5QNSUfiehiCvrnIP8KyZRLDoda1SN1YIsRiHbNkTwvgdnnDFRPq13M0wVdhUspd7iSG9tF8eJOElhXGpsEM21b9owsa+c5SxPQqwUnB70tzER+DlFgerdSnqlLBZrKltAPmaCs1c8Y5RxKUlYIof5plXa0A3mAu8aIm3lF8zWWPC9HKJ7+xViOmLGurk9wD+qhsQN6N2BFiJNeVRPfYBFPmhk7CjZQBpS+QZ/xaRHTbSGf+Ws6uhnReBO5QDgpbsCiKpfPkX6v5hoWQkelxnmG7QklAjlv4+CV5yE9ThuaN6YRIK5tMhKpCrx0RAD/x5iK0JpFWA9Fy6cxXCxuGLowUk79AAwYw86ltgD4wvEHgAGWjiUyUg3tZsNePCOiJfG859q4d55ujXf6XVcNPZVI5FWvTQbDS3VPKgVrUqvDbkkY1j83sI9YzkPtCM66QvEIyxaS025+CY3HFMkyKadj204a9VvjBQrteIUtjl1vQ5VpZlaq8NWbcZpBA6N7PEck3SN9cxdZn9DS1ktW5QiSqJEPgdSB0ukfRW8RJajqe5YkzBUc/gcy5yDlsV8iqBUOVC45Pk9b24n5tiKqOunnrBVo6xJ2rkDXgvXvp7oTLqGrUknbXW2WmX4XJZGR1P80prfl4yHmPeDMvSnhSr/pd9vyn8t4HZOLQXfGX+lDMlU7zzliSBrfBsJ/f1J//+U3cFHJnfXiLqHRDqaq+Qg1wtLzsPyP6Tnd5TFDnCeCRyw7chq4TnAQQM46SWVqarAzEFqw+Q8JCuOFJyByLlcTM59wkkQWzPv7eAKRh8JrGcWK8G4Og7dLmKD7hr+qYiZ5X2aILrPSNZzZ8czZpaixFpZpNOiryu+qhcUr4S1XvyR4wKF6beHN98kHShfdmkDnnDK5D4kiX7O652lPApFRdg16y5JiCCIWi2cUqwNvDIU73D/nGVkbASvuq487yPXVfQYTPqOSx4fti8/ot3j6Gn9tW+9r3jnUXwo918GgeDUo6o0kqQZPoGrLC2vnlZkp/j1qdrY+3UMN0hzpsP7cvMj39zc/BQ/c1Okxs9GNvrrngvQP2pv5W1XLtBCPrwa95Mrce9+ct/l3gU3Rn6VUS5OPvwk39jZw1sjH1yJ/MEn+UbRdz+OfGuGv9yxQ18aZK9YBLEeEeR8gxdEFLvEUvSDRFHxNomSz+Tc6iIezoeOSUno4XE4qIG33KAZXBy/POucYiYm7gPbRUR9ZjvrIsJMr88BS81Tn9oRn4BxEyctOv0S00Zq+yNUi0LO9F0JnEOnXnHFPVzXqvZCm0wWCwi1XH9kdOFl2ASdAgrG3p3T/EFzK21jt3svfQK5snn4gFhehR6+wsL7fwA=&lt;/diagram&gt;&lt;/mxfile&gt;">
```

Comment:
there's some weird green overlay and text in this diagram?

Reply:
Yes, Stas also complained, let's please talk about my idea :)

Reply:
removed it